code/workspaces/test/test-script/10893_STS.md

```diff
@@ -200,13 +214,25 @@ System testing is carried out as part of every sprint to ensure the completed st
 >>> | 5 | Set a sharing status and press create | The notebook is successfully created |  |
 ---
 
+>>>###### **TC2.3.5 Extra fields when creating R Studio notebook**
+>>> | **JIRA ID(s):** | [NERCDL-683](https://jira.ceh.ac.uk/browse/NERCDL-683?jql=key%20in%20(NERCDL-683)) |OVERALL RESULT: |  |
+>>> | ------------ | --------- | --------- | ------|
+>>> | **Number** | **Step** | **Expected Result** | **Result** |
+>>> | 1 | Navigate to the notebooks page | Pre-existing notebooks are listed here |  |
+>>> | 2 | Click on the create button | A dialog is shown with the following fields: Display Name, Notebook Type, URL, Data Store, Source Path, Description and Sharing Status |  |
+>>> | 3 | Fill out the required fields. Set the typ to R studio | And extra field appears when R Studio is selected. This is a version field. 4.0.1 is the default value |  |
+>>> | 4 | Click on the version field | A dropdpwn with two options is displayed: 4.0.1 & 3.5.3 |  |

```

Comment:
I assume the text in here is the same as in the other file so any typos etc. apply to this file too